src/common/collection_column_specs/parse_sample_template.py

```diff
@@ -0,0 +1,172 @@
+"""
+`parse_sample_spec` is a function designed to process a sample template YAML file and a core YAML file obtained from
+the Sample Service. It transforms the sample template entries into a column specifications format, making them
+suitable for use by the Collection Service.
+
+The sample template file can be downloaded from
+https://github.com/kbase/sample_service_validator_config/tree/master/templates
+The core yaml file can be downloaded from
+https://github.com/kbase/sample_service_validator_config/blob/master/vocabularies/core.yml
+
+PLEASE NOTE:
+users should manually examine the output yaml file and make sure the output is correct
+then manually copy the output to src/common/collection_column_specs/samples-[collection].yml
+In other words, the output of this script is to save the user's time when constructing the sample column
+specifications and is not expected to be completely accurate.
```

Comment:
Can you edit this slightly (e.g. remove the function reference) and stick it in the parser description?

Reply:
👍

Reply:
I'd include the stuff about the download locations and the PLEASE NOTE part too

Reply:
👍